### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by f3l1x** [juicyfx/now-php#3]
- next-gen PHP builder ✅
- Composer
- config.composer: true ✅
- composer.json detection ✅
- zero config ✅
- now dev ✅
- rewrite to typescript ✅

--- 

Feel free to suggest any feature you like.

**Comment by ties-v** [comment]
Custom php.ini, see #11

**Comment by f3l1x** [comment]
> Custom php.ini, see #11

Included in the features. ;-)

**Comment by AugustinMauroy** [comment]
In the examples/tests, it would be very interesting to see how wordpress works.

**Comment by AugustinMauroy** [comment]
I understand your point of view. But the thing is, wordpress represents a big chunk of the php that's run around the world.

**Comment by f3l1x** [comment]
Yep, but also there are some hard limits, like 50MB of lambda/function size. Can you please collaborate on this a prepare some kick-off?
